Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -150,7 +150,10 @@ class ActivityStreamPanel: UICollectionViewController, HomePanel {
     func applyTheme() {
         collectionView?.backgroundColor = UIColor.theme.browser.background
         topSiteCell.collectionView.reloadData()
-        reloadAll()
+        if let collectionView = self.collectionView {
```

Comment:
This doesn't crash: ``` if let collectionView = self.collectionView, collectionView.numberOfSections > 0, collectionView.numberOfItems(inSection: 0) > 0 {```

We should probably get a super-review from @farhanpatel. I am not sure why  indexPathsForVisibleItems would be invalid when first shown at startup.